packages/editor/src/extensions/code-block/highlighter.ts

```diff
@@ -221,8 +241,6 @@ export function HighlighterPlugin({
                   decorations = decorations.remove(toRemove);
                 if (toAdd.length > 0)
                   decorations = decorations.add(tr.doc, toAdd);
-              } else {
-                languages[id] = language;
```

Comment:
Why are you flipping the `if` statement? We don't do negative checks in the if statement if possible.

Reply:
Ok I will flip it back and it will make more sense.